app/sentry.ts

```diff
@@ -24,3 +31,25 @@ Sentry.init({
     },
     ...config.sentry
 });
+
+Sentry.configureScope(function(scope) {
+    function addAdapterError(error: any, event: Sentry.Event) {
+        if (error.isAdapterError) {
+            event.extra = {
+                ...event.extra,
+                adapter_errors: error.errors,
+                adapter_errors_json: JSON.stringify(error.errors)
+            }
+        }
+    }
+
+    scope.addEventProcessor(function(event: Sentry.Event, hints: Sentry.EventHint) {
+        addAdapterError(hints.originalException, event);
+
+        const args: any[] = event.extra?.arguments || [];
+        for (const arg of args) {
+            addAdapterError(arg, event);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114150)